src/containers/HeadContainer.vue

```diff
@@ -1,23 +1,34 @@
 <template>
-    <nav v-if="loggedIn">
-        <div v-if="userType === 'doctor'">
-            
-        </div>
-        <div v-else>
-            
-        </div>
-    </nav>
-    <Toast /> 
+  <nav v-if="!!userType">
+    <Button label="Log out" @click="userLogout" />
+    <div v-if="userType === 'doctor'">doctor</div>
+    <div v-else>patient</div>
+  </nav>
+  <Toast />
 </template>
 
 <script setup>
 import Toast from "primevue/toast";
-import Cookies from 'js-cookie'
+import Button from "primevue/button";
 
-const userType = Cookies.get('user-type')
-const loggedIn = Cookies.get('logged-in')
+import Cookies from "js-cookie";
+import { logout } from "../services/api";
+import { ref, watch } from "vue";
+import { useRouter, useRoute } from 'vue-router'
+
+const router = useRouter();
+const route = useRoute();
+const userType = ref(Cookies.get('user-type'));
```

Comment:
Use `useUserSession` service with `clearUserSession` function instead

Reply:
`function logout() {
  clearUserSession();
  userSession.token = "";
  userSession.userType = "";
  router.push({ name: 'login' });
}`

Do I have to clear token and userType llike that?

Reply:
Yes but I'd move it directly to the clearUserSession() function so this logout function would look like:
`function logout() { clearUserSession(); router.push({ name: 'login' }); }`

Reply:
I inserted cleaning user token and type in the useUserSession file.